.github/workflows/integration-test-config-latest-k8s/action.yaml

```diff
@@ -13,10 +13,25 @@ runs:
       with:
         go-version: "1.23"
 
-    - name: Setup Minikube
+    # - name: Setup Minikube
```

Comment:
Since this change is switching from minikube to kind, how about changing the name to not mention minikube anymore?

Suggestion: 'name: Setup k8s cluster'